src/components/Button/Button.module.scss

```diff
@@ -26,6 +26,21 @@ $disabled-on-primary-color-opacity: 0.5;
   /* Will be updated dynamically after render by js */
   --element-width: 32;
   --element-height: 32;
+  min-width: var(--element-width);
```

Comment:
Moved from line 50

Reply:
I don't see the effect of this. are you sure it's working? it's missing the `px`

Reply:
in addition, the `element-width` will overridden after being calculated in the js

Reply:
Oh yeah your'e right, the definition is missing the `px`, it is being overridden with JS (`Button.tsx:162-163`) so it's used as a default/fallback value, I will add `px` to the definition of the variable

Reply:
It did not work before as the fallback value of 32px as the value was illegal, now when I changed it to be a valid value if messes up some component that use Button with a smaller width than 32px, so I think we should remove the default value at all